.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,34 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: triage
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
```

Comment:
one more thing: the bugs they put here should be specific to the Python Google Ads API lib only. I suggest we put pointers in to the AdWords API lib as well as the forum and support email alias for general Ads API questions